### ISSUE TITLE
node >= 10.13.0 required

### ISSUE THREAD

**Issue by patsevanton** [verdaccio/ui#402]
Hello!

> Unsupported engine for @verdaccio/ui-theme@0.3.9: wanted node >= 10.13.0. Current node 8.17.0

cat /etc/redhat-release 
```CentOS Linux release 7.6.1810 (Core) ```

node -v
```v8.17.0```

npm install -g verdaccio
```
/usr/bin/verdaccio -> /usr/lib/node_modules/verdaccio/bin/verdaccio
npm WARN notsup Unsupported engine for @verdaccio/ui-theme@0.3.9: wanted: {"node":">= 10.13.0","npm":">=5"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm WARN notsup Not compatible with your version of node/npm: @verdaccio/ui-theme@0.3.9

+ verdaccio@4.4.0
```

![image](https://user-images.githubusercontent.com/10828883/71556602-16c08280-2a65-11ea-9ad7-ce83bd4fc5c9.png)

**Comment by DanielRuf** [comment]
I wonder why we don't test Node 8 anymore:

https://github.com/verdaccio/verdaccio/blob/master/.circleci/config.yml#L15

cc @juanpicado

**Comment by DanielRuf** [comment]
https://github.com/verdaccio/verdaccio/pull/1495

cc @sergiohgz

**Comment by DanielRuf** [comment]
So our docs are also outdated as we require npm >= 5 too.

https://verdaccio.org/docs/en/installation#prerequisites

**Comment by patsevanton** [comment]
May be change https://verdaccio.org/docs/en/installation#prerequisites  ?

**Comment by DanielRuf** [comment]
Mo, that change should not have happened actually as Node 10 requirement was planned for verdaccio 5.

We follow SemVer and verdaccio 4 itself requires Node >= 8. verdaccio 5 should require Node >= 10.

Also see https://github.com/verdaccio/verdaccio/pull/1335 and https://github.com/verdaccio/verdaccio/blob/master/package.json#L136

**Comment by DanielRuf** [comment]
No, that change should not have happened actually as Node 10 requirement was planned for verdaccio 5.

We follow SemVer and verdaccio 4 itself requires Node >= 8. verdaccio 5 should require Node >= 10.

Also see https://github.com/verdaccio/verdaccio/pull/1335 and https://github.com/verdaccio/verdaccio/blob/master/package.json#L136

**Comment by DanielRuf** [comment]
Currently we have some mess here. We do not test on Node 8 anymore but say that Node 8 is supported. As there were more changes we should change that requirement to 10 now but this would require a major version bump.

**Comment by juanpicado** [comment]
I will revert it @DanielRuf .... I think the engine on this repo does not make sense, we ship a JS bundle for the browser.

**Comment by juanpicado** [comment]
We don't test here on Node 8 due is just for development, `url-loader` is a devDependency and I think develop at least in `>=v10` make totally sense, but the `engine` flag here should not be added or at least stay on `>=v8`

**Comment by juanpicado** [comment]
Shipped on `v4.4.1`
